external/CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+add_subdirectory("ebpf-verifier" EXCLUDE_FROM_ALL)
+add_subdirectory("Catch2" EXCLUDE_FROM_ALL)
+
+# Special target that we can link to external dependencies
+# to override build settings
+add_library("ebpf_for_windows_external_settings" INTERFACE)
+target_compile_options("ebpf_for_windows_external_settings" INTERFACE
+  "/w"
+)
+
+target_compile_definitions("ebpf_for_windows_external_settings" INTERFACE
+  "_CRT_SECURE_NO_WARNINGS"
+  "_SILENCE_ALL_CXX17_DEPRECATION_WARNINGS"
+)
+
+#
+# Catch2
+#
+
+target_link_libraries("Catch2" PRIVATE
+  "ebpf_for_windows_external_settings"
+)
+
+#
+# ebpfverifier
+#
+
+target_link_libraries("ebpfverifier" PRIVATE
+  "ebpf_for_windows_external_settings"
+)
+
+# Fix missing dependency between ebpfverifier and yaml-cpp
+add_dependencies("ebpfverifier"
+  "yaml-cpp"
+)
+
+# Fix missing INTERFACE include directories
+target_include_directories("ebpfverifier" INTERFACE
+  ebpf-verifier/src
+  ebpf-verifier/external/elfio
+)
+
+add_library("external::ebpfverifier" ALIAS "ebpfverifier")
+
+#
+# boost (downloaded by ebpfverifier)
+#
+
+add_library("boost" INTERFACE)
+target_include_directories("boost" INTERFACE
+  "${CMAKE_BINARY_DIR}/packages/boost/lib/native/include"
+)
+
+add_library("external::boost" ALIAS "boost")
+
+#
+# bpftool
+#
+
+add_executable("bpftool"
+  bpftool/src/common.c
+  bpftool/libbpf/src/hashmap.c
```

Comment:
reorder these two so lines are in alphabetical order

Same for any other places with a list of source files.

Reply:
When the CMakeLists.txt is opened as a project from a compatible editor/IDE (VSCode, Visual Studio, CLion, QtCreator, ...), the file explorer should automatically sort them alphabetically. I think it makes maintenance harder to manually sort the list in the file